packages/block-editor/src/components/rich-text/index.js

```diff
@@ -603,6 +607,11 @@ function RichTextWrapper(
 			fontFamily={ fontFamily }
 			fontWeight={ fontWeight }
 			fontStyle={ fontStyle }
+			minWidth={ minWidth }
+			maxWidth={ maxWidth }
+			onBlur={ onBlur }
+			setRef={ setRef }
+			id={ props.id }
```

Comment:
@geriux could you please add there a comment? It can be something similar to the [one](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-aztec/src/AztecView.js#L214) related to `styles`